### PR TITLE
fix(argo-cd): envFrom in repoServer

### PR DESCRIPTION
**Pull request by czerasz**
Fix typo introduced in https://github.com/argoproj/argo-helm/pull/743

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [ ] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [ ] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by mkilchhofer** [comment]
Oh good catch, I missed that in the last review.
Another thing that was missing in PR #743 is the documentation inside the charts README.md. Would you mind to catch up on this also? :-)

**Comment by czerasz** [comment]
@mkilchhofer documentation updated. Thank You for spotting this!

Not sure where to put this... Below an example how we use this:

```yaml
envFrom:
- configMapRef:
  name: proxy-config
```

**Comment by mkilchhofer** [comment]
> @mkilchhofer documentation updated. Thank You for spotting this!
> 
> Not sure where to put this... Below an example how we use this:
> 
> ```yaml
> envFrom:
> - configMapRef:
>   name: proxy-config
> ```

You can simply put `[] (See [values.yaml](values.yaml))` in the column `Default`. The value `clusterCredentials` also uses this notation :-)

**Comment by czerasz** [comment]
@mkilchhofer good tip! Followed Your advice :slightly_smiling_face:

**Comment by mkilchhofer** [comment]
> @mkilchhofer good tip! Followed Your advice 🙂

👍 
We need two `)` at the end

File: charts/argo-cd/README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ Helm v3 has removed the `install-crds` hook so CRDs are now populated by files i
 | controller.extraArgs | Additional arguments for the controller. A list of flags | `[]` |
 | controller.enableStatefulSet | Enable deploying the controller as a StatefulSet instead of a Deployment. Used for HA installations. | `false` |
 | controller.env | Environment variables for the controller. | `[]` |
+| controller.envFrom | `envFrom` to pass to the controller. | `[]` (See [values.yaml](values.yaml) |
```
Comment (mkilchhofer) [inline charts/argo-cd/README.md:150]:
```suggestion
| controller.envFrom | `envFrom` to pass to the controller. | `[]` (See [values.yaml](values.yaml)) |
```

File: charts/argo-cd/README.md [hunk 2]
```diff
@@ -198,6 +199,7 @@ Helm v3 has removed the `install-crds` hook so CRDs are now populated by files i
 | repoServer.containerPort | Repo server port | `8081` |
 | repoServer.extraArgs | Additional arguments for the repo server. A  list of flags. | `[]` |
 | repoServer.env | Environment variables for the repo server. | `[]` |
+| repoServer.envFrom | `envFrom` to pass to the repo server. | `[]` (See [values.yaml](values.yaml) |
```
Comment (mkilchhofer) [inline charts/argo-cd/README.md:202]:
```suggestion
| repoServer.envFrom | `envFrom` to pass to the repo server. | `[]` (See [values.yaml](values.yaml)) |
```

File: charts/argo-cd/README.md [hunk 3]
```diff
@@ -258,6 +260,7 @@ Helm v3 has removed the `install-crds` hook so CRDs are now populated by files i
 | server.containerPort | Server container port. | `8080` |
 | server.extraArgs | Additional arguments for the server. A list of flags. | `[]` |
 | server.env | Environment variables for the server. | `[]` |
+| server.envFrom | `envFrom` to pass to the server. | `[]` (See [values.yaml](values.yaml) |
```
Comment (mkilchhofer) [inline charts/argo-cd/README.md:263]:
```suggestion
| server.envFrom | `envFrom` to pass to the server. | `[]` (See [values.yaml](values.yaml)) |
```

File: charts/argo-cd/README.md [hunk 4]
```diff
@@ -343,6 +346,7 @@ Helm v3 has removed the `install-crds` hook so CRDs are now populated by files i
 | dex.metrics.serviceMonitor.selector | Prometheus ServiceMonitor selector. | `{}` |
 | dex.name | Dex name | `"dex-server"` |
 | dex.env | Environment variables for the Dex server. | `[]` |
+| dex.envFrom | `envFrom` to pass to the Dex server. | `[]` (See [values.yaml](values.yaml) |
```
Comment (mkilchhofer) [inline charts/argo-cd/README.md:349]:
```suggestion
| dex.envFrom | `envFrom` to pass to the Dex server. | `[]` (See [values.yaml](values.yaml)) |
```

File: charts/argo-cd/README.md [hunk 5]
```diff
@@ -376,6 +380,7 @@ through `xxx.extraArgs`
 | redis.extraArgs | Additional arguments for the `redis-server`. A list of flags. | `[]` |
 | redis.name | Redis name | `"redis"` |
 | redis.env | Environment variables for the Redis server. | `[]` |
+| redis.envFrom | `envFrom` to pass to the Redis server. | `[]` (See [values.yaml](values.yaml) |
```
Comment (mkilchhofer) [inline charts/argo-cd/README.md:383]:
```suggestion
| redis.envFrom | `envFrom` to pass to the Redis server. | `[]` (See [values.yaml](values.yaml)) |
```

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**